src/components/publishing/sections/fullscreen_viewer/__test__/__snapshots__/fullscreen_viewer.test.tsx.snap

```diff
@@ -140,7 +140,7 @@ exports[`renders properly 1`] = `
   height: 100vh;
   overflow: hidden;
   position: fixed;
-  z-index: 20;
+  z-index: 1070;
```

Comment:
This _seems_ arbitrary, but it's actually the z-index we use in Force for modals: https://github.com/artsy/force/blob/master/desktop/components/stylus_lib/index.styl#L40. Maybe worth putting in a comment.

Reply:
Maybe instead of a comment, this could go into a central file that’s all about styling constants and name the variable something like “modal z-index” so it’s self-documenting. I.e. like that file you linked to in Force.

Reply:
I'm going to table this for now and eventually get real (typed) system for these. There's also the `faux-underline` CSS that could be nice to include.

Reply:
👍